### PR TITLE
bfb-install: add cleanup code for runtime update

### PR DESCRIPTION
**Pull request by lsun100**
This commit adds cleanup code for runtime update, so SP2.BIT2 could be always cleared when script completes.

### CODE REVIEW

File: scripts/bfb-install [hunk 1]
```diff
@@ -196,6 +196,8 @@ push_boot_stream_via_local_rshim()
     clear_boot_fifo
     # Set SP2.BIT2=1
     ${BF_REG} $(basename ${rshim_node}) 0x13000c48.64 0x4 >/dev/null
+    # Set SWINT2.BIT2
+    ${BF_REG} $(basename ${rshim_node}) 0x13000318.64 0x4 >/dev/null
```
Comment (pgeng-nv) [inline scripts/bfb-install:200]: We may want to add BF_REG to the global variable list line above:

    # Global variables used:
    #   $bfb, $cfg, $rootfs, $pv, $rshim_node, $sudo_prefix

However, I tend to agree it's a bit of work to update this list on an ongoing basis, and its usefulness may be limited. So we could remove just remove these two comment lines instead.

File: scripts/bfb-install [hunk 2]
```diff
@@ -459,6 +461,17 @@ cleanup() {
...
+  if [ $runtime -eq 1 ]; then
+    # Reset to default state.
+    echo "BOOT_RESET_SKIP 0" > ${rshim_node}/misc
+
+    # Cleanup SP2.BIT2 and trigger SWINT2.
+    sp2=`${BF_REG} $(basename ${rshim_node}) 0x13000c48.64 | awk '{print $3}'`
+    sp2=$((sp & ~4))
+    ${BF_REG} $(basename ${rshim_node}) 0x13000c48.64 $sp2 >/dev/null
+    ${BF_REG} $(basename ${rshim_node}) 0x13000318.64 0x4 >/dev/null
+  fi
```
Comment (pgeng-nv) [inline scripts/bfb-install:474]: Same comment as the first one. It applies to the newly added variable $runtime, $BF_REG etc. We can remove the two function comment lines.

File: scripts/bfb-install [hunk 3]
```diff
@@ -459,6 +461,17 @@ cleanup() {
...
     fi
   fi
+
+  if [ $runtime -eq 1 ]; then
+    # Reset to default state.
+    echo "BOOT_RESET_SKIP 0" > ${rshim_node}/misc
+
+    # Cleanup SP2.BIT2 and trigger SWINT2.
+    sp2=`${BF_REG} $(basename ${rshim_node}) 0x13000c48.64 | awk '{print $3}'`
+    sp2=$((sp & ~4))
```
Comment (pgeng-nv) [inline scripts/bfb-install:471]: The `sp` in line 471 should be `sp2`?

Comment (lsun100) [reply scripts/bfb-install:471]: good catch, thanks, fixed.

File: scripts/bfb-install (hunk 2 above)
Comment (lsun100) [reply scripts/bfb-install:474]: done

File: scripts/bfb-install (hunk 1 above)
Comment (lsun100) [reply scripts/bfb-install:197]: done

**Review by pgeng-nv — APPROVED** [review]

**Pull request merged**